packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -41,15 +48,29 @@ - (instancetype)initWithFrame:(CGRect)frame
               binaryMessenger:(NSObject<FlutterBinaryMessenger>*)messenger {
   if ([super init]) {
     _viewId = viewId;
-    _webView = [[WKWebView alloc] initWithFrame:frame];
+
     NSString* channelName = [NSString stringWithFormat:@"plugins.flutter.io/webview_%lld", viewId];
     _channel = [FlutterMethodChannel methodChannelWithName:channelName binaryMessenger:messenger];
+    _javaScriptChannelNames = [[NSMutableSet alloc] init];
+
+    WKUserContentController* userContentController = [[WKUserContentController alloc] init];
+    if (![args[@"javascriptChannelNames"] isKindOfClass:[NSNull class]]) {
+      NSArray* javaScriptChannelNames = args[@"javascriptChannelNames"];
```

Comment:
This is really close to being the same code as `onAddJavascriptChannels`. Would it make sense to change the method signature of that class and make a call to it here instead?